### DISCUSSION TITLE
No module named 'rembg'

### DISCUSSION THREAD

**Discussion by EgorSHep** [vladmandic/sdnext#2895] [Q&A]
When trying to remove the background in rembg, I got the following message: Rembg load failed: No module named 'rembg'.  How to fix it?
![image](https://github.com/vladmandic/automatic/assets/128888820/e34e3a27-a727-4721-a7ad-0de46dfe1cf6)

**Comment by vladmandic** [comment]
need far more details than that. and if its an issue, open an issue and provide all necessary details - this is hardly a discussion topic.

**Reply by EgorSHep** [reply]
I installed the latest version of SD.Next. After running it, I go to the Process tab, open the image, select Rembg and click Generate ....
then I get an error in the console "Rembg load failed: No module named 'rembg'".
![image](https://github.com/vladmandic/automatic/assets/128888820/5a08db3a-08f6-4775-b261-5124a06eeb85)
Am I doing something wrong?

**Comment by vladmandic** [comment]
fixed in latest dev

**Comment by aivrar** [comment]
I have this problem too, I have latest git also. [edit] I get this trying to create video.
